docs/guide/user_guide/intro_getting_started.rst

```diff
@@ -0,0 +1,99 @@
+.. _intro_getting_started:
+
+***************
+Getting Started
+***************
+
+Now that you have read the :ref:`installation guide<installation_guide>` and installed ansible-pylibssh on a your system.
+
+.. contents::
+   :local:
+
+Checking libssh version
+=======================
+
+.. code-block:: python
+
+   from pylibsshext._libssh_version import LIBSSH_VERSION
+
+   print(LIBSSH_VERSION)
```

Comment:
Instead of putting Python sources in RST, let's put them in Python files and just include their contents (full or partial). It's easier to maintain files with complete sources that are just included in guides in parts. And it's possible to lint/check those files too, unlike code blocks in sphinx.

Reply:
(you may delegate this to me if you want as I have a lot of experience using the inclusions)